### ISSUE TITLE
blackbox_register_new_file stopped working after eab0418f

### ISSUE THREAD

**Issue by harrisonpage** [StackExchange/blackbox#104]
Still trying to wrap my head around this problem. 

After doing a git pull, running blackbox_register_new_file shows this error:

```% echo nevermind > whatever.txt```
```% blackbox_register_new_file whatever.txt```
```gpg: whatever.txt: encryption failed: No public key```

I double-checked that all public keys had subkeys which were not expired. 

When I step back from the latest release to eab0418f the command works.

Adding "set -x" to the top of blackbox_register_new_file reveals this output:

```+ gpg --use-agent --yes --trust-model=always --encrypt -o whatever.txt.gpg '-rRECIPIENT1
-rRECIPIENT2
-rRECIPIENT3' whatever.txt
gpg: RECIPIENT1
-rRECIPIENT2
-rRECIPIENT3: skipped: No public key
gpg: whatever.txt: encryption failed: No public key```

In release eab0418f and before, output looks like this:

```+ gpg --yes --trust-model=always --encrypt -o whatever.txt.gpg -rRECIPIENT1  -rRECIPIENT2 -rRECIPIENT3 whatever.txt```

All the recipients are on the same line. 

What baffles me is that the code in _blackbox_common.sh is the same between the two versions:

```gpg --yes --trust-model=always --encrypt -o "$encrypted" $(awk '{ print "-r" $1 }' < "$BB_ADMINS") "$unencrypted"```

Anybody else seeing this error?

**Comment by selfcommit** [comment]
Didn't mean to close this without confirming.  Please try the most recent build to see if this is resolved for you.

**Comment by selfcommit** [comment]
@harrisonpage - I figured your issue would be resolved by 7d9c07678bc1266c6de03c999b796c8d104ea1d6  But it looks like you're not effected by the extra quotes.  Can you pull the lastest release and let me know if you're still seeing this issue?

**Comment by harrisonpage** [comment]
@selfcommit Awesomes: It is fixed! Thanks for following up.
